apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/familieforhold/RedigerFamilieforhold.tsx

```diff
@@ -55,13 +55,18 @@ export const RedigerFamilieforhold = ({ behandling, personopplysninger }: Props)
     if (foreldre.length != 2) {
       setFeilmelding('Mangler en eller flere forelder')
```

Comment:
```suggestion
    if (foreldre.length !== 2) {
      setFeilmelding(`Det er angitt ${foreldre.length} foreldre, men det skal være 2.`)
```

eller noe slikt? 

burde vi ha noe for ukjent forelder?

Reply:
Tenkte ikke på ukjent foreldre nei

Reply:
Burde ikke ha valideringen egentlig da?

Reply:
Krever minst 1 avdød. Da tillater vi ukjent gjenlevende men ukjent avdød vil kreve overstyrt trygdetid og hvis det ikke er en manuell migrering som har valgt det ved opprettelse så er det for sent.